7-extract_plier_pathways.R

```diff
@@ -293,28 +325,26 @@ for (seed_index in 1:length(norm.train.files)) {
   # renames list levels
   names(plier_results_list) <- perc_seq
   for (i in perc_seq) {
-    names(plier_results_list[[i]]) <- norm_methods
+    if (i %in% c("0", "100")) {
+      names(plier_results_list[[i]]) <- norm_methods_if_0_100
+    } else {
+      names(plier_results_list[[i]]) <- norm_methods_else
+    }
   }
   
-  # write test file
+  # Check for failure to converge, and set to NA
+  plier_results_list <- purrr::modify_depth(plier_results_list, 2,
+                                            check_plier_failure_to_converge
+  )
   
-  write_rds(x = plier_results_list,
-            path = str_c("plier.", seed_index, ".rds"))
```

Comment:
Just checking that no longer writing the PLIER results as an RDS was intentional?